utils/man/scoutfs.5

```diff
@@ -197,6 +197,15 @@ name, total value, and a count of contributing attributes can be read
 with the
 .IB READ_XATTR_TOTALS
 ioctl.
+.TP
+.B .worm.
+Attributes with the .worm. flag are used to maintain the worm policy
```

Comment:
Let's also mention: the value is a sec.nsec timeval in wall clock GMT and that the hidden tag must also be used so that the worm attribute is kept private to this scoutfs volume.